bugbug/bug_features.py

```diff
@@ -179,6 +179,9 @@ def __init__(self, feature_extractors, commit_messages_map=None):
     def fit(self, x, y=None):
         return self
 
+    def cleanup(self, text):
+        return re.sub(r'http\S+', 'URL', text)
```

Comment:
This will also match invalid URLs, but it should be fine.

Reply:
I'd define, outside of this class, a list of "text cleanup" classes (just like the classes we have for feature extraction). Then you call them either in this `cleanup` function, or in `transform`.